index.txt

```diff
@@ -1,4 +1,4 @@
-Libro1: Introducción
+Libro 1: Introducción
 Libro 2: Fundamentos
 Libro 3: Metodología
-Libro 4: Aplicaciónes
\ No newline at end of file
+Libro 4: Aplicaciónes
```

Comment:
Typo, should be "Aplicaciones"

Reply:
Great point! Merging to main branch, performing fast forward :)